app/models/customer.rb

```diff
@@ -0,0 +1,9 @@
+class Customer < ApplicationRecord
+  has_many :customer_items
+  has_many :items, through: :customer_items
+
+  def add_item(item_id)
```

Comment:
See my comment in your Customer controller. I don't think this method will be necessary once you make the update that I suggested.